src/metrics.jl

```diff
@@ -99,8 +99,13 @@ struct MeanSqDeviation <: SemiMetric end
 struct RMSDeviation <: Metric end
 struct NormRMSDeviation <: Metric end
 
+const RealAbstractArray{T <: Real} =  AbstractArray{T}
 
-const UnionMetrics = Union{Euclidean,SqEuclidean,Chebyshev,Cityblock,TotalVariation,Minkowski,Hamming,Jaccard,RogersTanimoto,CosineDist,CorrDist,ChiSqDist,KLDivergence,RenyiDivergence,BrayCurtis,JSDivergence,SpanNormDist,GenKLDivergence}
+struct PeriodicEuclidean{W <: RealAbstractArray} <: Metric
```

Comment:
```suggestion
struct PeriodicEuclidean{W <: AbstractArray{<: Real}} <: Metric
```
Then you can drop `RealAbstractArray` above.

Reply:
Sure, I just copied it from the `wmetrics.jl`.